<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="39259" started-at="2020-09-06T17:25:44 IST" finished-at="2020-09-06T17:26:23 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="39259" started-at="2020-09-06T17:25:44 IST" finished-at="2020-09-06T17:26:23 IST">
      <class name="wix.test.cases.pollsTestCases">
        <test-method status="PASS" signature="setUp()[pri:0, instance:wix.test.cases.pollsTestCases@5891e32e]" name="setUp" is-config="true" duration-ms="37948" started-at="2020-09-06T17:25:44 IST" finished-at="2020-09-06T17:26:22 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="Wavity_Login_Logout()[pri:0, instance:wix.test.cases.pollsTestCases@5891e32e]" name="Wavity_Login_Logout" duration-ms="138" started-at="2020-09-06T17:26:22 IST" finished-at="2020-09-06T17:26:22 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at CommonComponents.Common.login(Common.java:20)
at wix.test.cases.pollsTestCases.Wavity_Login_Logout(pollsTestCases.java:42)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Wavity_Login_Logout -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:wix.test.cases.pollsTestCases@5891e32e]" name="tearDown" is-config="true" duration-ms="900" started-at="2020-09-06T17:26:22 IST" finished-at="2020-09-06T17:26:23 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- wix.test.cases.pollsTestCases -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
